### DISCUSSION TITLE
Error instalacion npm install -g ganache

### DISCUSSION THREAD

**Discussion by IvoMaciel** [smartcontractkit/full-blockchain-solidity-course-py#1907] [General]
Error installing npm install -g ganache
When I install it sudo npm install -g ganache I get the following message:
roberto@DESKTOP-RQNJBQS:/demos/web3_py_simple_storage$ sudo npm install -g ganache

I changed 338 packages and audited 366 packages in 21 seconds

6 packages are seeking funding
run `npm fund` for more details

15 vulnerabilities (1 low, 2 moderate, 11 high, 1 critical)

To fix issues that don't require attention, run:
npm audit fix

To fix all issues (including breaking changes), run:
npm audit fix --force

Run `npm audit` for more details.

**Comment by ghost** [comment]
### Steps to resolve:
1. **Install `ganache` anyway**: 
   It seems `ganache` installed correctly despite the vulnerabilities. You can check whether it's working by running:
   ```bash
   ganache --version
   ```
   If that works, the installation was successful, and the vulnerabilities are simply related to other packages in the global `node_modules`.

2. **Fixing vulnerabilities**:
   - If you'd like to address the vulnerabilities, you can follow the recommendation in the message by running:
     ```bash
     npm audit fix
     ```
     This will attempt to fix non-breaking vulnerabilities automatically.

   - If the audit fix doesn't resolve everything or you want to fix even breaking changes, use:
     ```bash
     npm audit fix --force
     ```
     **Note**: Running this with `--force` might update packages in a way that could break compatibility with others. Use this only if you're not worried about compatibility issues.

3. **Check for vulnerabilities**:
   You can manually inspect the vulnerabilities by running:
   ```bash
   npm audit
   ```
   This will give you more details on what the vulnerabilities are and help you decide whether they require immediate attention.

If `ganache` is working fine after installation and you're not concerned about the vulnerabilities in the global packages (as long as it doesn't affect `ganache`), you can continue developing.
